### DISCUSSION TITLE
Does renovate update existing MRs with same branchPrefix ?

### DISCUSSION THREAD

**Discussion by alreadyhaveanaccount4** [renovatebot/renovate#20693] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

suppose there are already branches with prefix *update/* that have nothing to do with renovate. When I now set *branchPrefix* also to *update/*, what would renovate do to these branches? Does It overwrite/delete them? 
In general - is it *unsafe* to have renovate use the same *branchPrefix* like the other branches in the Project? I've seen warnings only for *branchName*.

### What you tried so far.

general question

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
If it has a branch name collision, it's usually going to force push over it. This is why I'd always recommend a unique branchPrefix

**Reply by alreadyhaveanaccount4** [reply]
Thanks for the quick reply.
So running renovate with branches *update/my-other-features* and *update/dep-updates* would not overwrite anything I guess?

**Reply by rarkins** [reply]
Your only concern is full branch name collisions. Renovate won't take over everything with the same prefix

**Reply by alreadyhaveanaccount4** [reply]
Alright, thanks!
